client/my-sites/posts/main.jsx

```diff
@@ -34,21 +34,11 @@ import {
 import { getEditorNewPostPath } from 'state/ui/editor/selectors';
 import { warningNotice } from 'state/notices/actions';
 import {
-	getSiteAdminUrl,
 	getSiteSlug,
 	isJetpackSite,
-	siteHasMinimumJetpackVersion
 } from 'state/sites/selectors';
 
 const PostsMain = React.createClass( {
-	componentWillMount() {
-		this.setWarning( this.props );
-	},
-
-	componentWillReceiveProps( nextProps ) {
-		this.setWarning( nextProps );
```

Comment:
👍 Hi @sirreal thank you for having a look into this.  You are tottaly right if I change this to:
		if ( nextProps.siteId !== this.props.siteId || nextProps.hasMinimumJetpackVersion !== this.props.hasMinimumJetpackVersion ) {
			this.setWarning( nextProps );
		}

Things work perfectly. Should I create a new PR with this change?